src/MicrosoftAuthFlow.js

```diff
@@ -121,17 +123,16 @@ class MicrosoftAuthFlow {
       return await retry(async () => {
         const msaToken = await this.getMsaToken()
 
-        if (options.doSisuAuth) {
-          assert(options.authTitle !== undefined, 'Please specify an "authTitle" in Authflow constructor when using sisu authentication')
+        if (options.flow === 'sisu') {
           debug(`[xbl] Sisu flow selected, trying to authenticate with authTitle ID ${options.authTitle}`)
           const deviceToken = await this.xbl.getDeviceToken(options)
           const sisu = await this.xbl.doSisuAuth(msaToken, deviceToken, options)
           return sisu
         }
 
-        const userToken = await this.xbl.getUserToken(msaToken, !options.authTitle)
+        const userToken = await this.xbl.getUserToken(msaToken, !this.doTitleAuth)
```

Comment:
this probably made more sense with flow == 'msa'.
side note I'm not a fan of the name "msa" in the enum but it's OK

Reply:
Ahah will revert it back and now you mention it maybe msal would be a better name.